rambo/cli.py

```diff
@@ -104,10 +114,10 @@ def destroy_cmd(ctx):
               help="Export all of %s's project code." % PROJECT_NAME.capitalize())
 @click.option('-O', '--output-path', type=click.Path(), default=None,
               help='The optional output path.')
-def export_cmd(force, src, output_path):
+def export_cmd(src, output_path, force):
```

Comment:
Probably don't want to leave "@cli.command('export'..." here since it's pretty much being replaced by the createproject command.